content/posts/move-over-ctf-writeup/index.md

```
@@ -0,0 +1,250 @@
+}
+
+public fun add_liquidity(..., liquidity_amount: u128, tick: u64, ...) {
+    let sqrt_price = tick_to_sqrt_price(tick);
+    let cost = (liquidity_amount * sqrt_price) / SCALE;
+    assert!((deposit as u128) >= cost, 0);
+}
+```
+
+`SCALE = 2^64`，當 `tick = 0` 時 `sqrt_price = 1`：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)